bundles/best_practices.rst

```diff
@@ -166,6 +166,70 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
+    cache:
+        directories:
+            - $HOME/.composer/cache/files
+    env:
+        global:
+            - TEST_COMMAND="./vendor/bin/simple-phpunit"
+
+    matrix:
+        fast_finish: true
+        include:
+              # Minimum supported PHP and Symfony version
+            - php: 7.2
```

Comment:
should this be 7.0 ? or is the comment wrong?

Reply:
Wrong comment. Thank you